docs/install-smart-home.md

```diff
@@ -0,0 +1,148 @@
+
+  <robo-wiki-grid-element-wrapper textAlign="center" :columns="3" flexible>
+    <robo-wiki-grid-element>
+      <robo-wiki-picture src="home-assistant/need_2.png" /> 
+      <b>Raspberry Pi 4 (at least 2 GB RAM)</b>
+    </robo-wiki-grid-element>
+    <robo-wiki-grid-element>
+      <robo-wiki-picture src="home-assistant/need_3.png" /> 
+      <b>SD card 16Gb+</b>
+    </robo-wiki-grid-element>
+    <robo-wiki-grid-element>
+      <robo-wiki-picture src="home-assistant/need_7.png" /> 
+      <a href="https://www.zigbee2mqtt.io/information/supported_adapters.html" target="_blank"><b>Zigbee adapter(Optionally)</b></a>
+    </robo-wiki-grid-element>
+  </robo-wiki-grid-element-wrapper>
+
+  <robo-wiki-grid-element-wrapper textAlign="center" :columns="2">
+    <robo-wiki-grid-element>
+      <robo-wiki-picture src="home-assistant/need_5.png" />
+      <a href="https://www.zigbee2mqtt.io/supported-devices/" target="_blank"><b>Zigbee smart devices(Optionally)</b></a>
+    </robo-wiki-grid-element>
+    <robo-wiki-grid-element>
+      <robo-wiki-picture src="home-assistant/need_9.png" />
+      <b>Desktop for setup</b>
+    </robo-wiki-grid-element>
+  </robo-wiki-grid-element-wrapper>
+
+
+## 1. Install Pre-requirements
+
+Robonomics docker contains:
+- Home Assistant
+- IPFS
+- MQTT Broker and Integration
+- Zigbee2MQTT
+- libp2p proxy
+- Robonomics Integration
+
+This article will show the installation process on ubuntu system. First you need to install next packages:
+
+<code-helper copy>
+
+```
+sudo apt-get install wget unzip git
+```
+</code-helper>
+
+Then you need to install Docker on PC. Installation instruction find on [the official website](https://docs.docker.com/engine/install/).
+
+<robo-wiki-note type="warning" title="Important information">
+
+  Add your user to docker group, to start docker containers without root permissions. Find [instruction here](https://docs.docker.com/engine/install/linux-postinstall/).
+
+</robo-wiki-note>
+
+## 2. Configure
+
+Download GitHub repository and go inside it:
+
+<code-helper copy>
+
+```
+https://github.com/PinoutLTD/home-assistant-web3-build.git
+cd home-assistant-web3-build/
+```
+</code-helper>
+
+then you have to create `.env` file. Convert it from `template.env` file:
+
+<code-helper copy>
+
+```
+mv template.env .env
+```
+
+</code-helper>
+
+After that,You may open the file and edit default values such as: 
+- Versions of packages
+- path to repository where will be stored all configurations folders.
+- time zone in ["tz database name"](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+
+## 3. Start 
+
+Run bash script and wait until the script will install all required packages:
+
+<code-helper copy>
+
+```
+bash setup.sh
+```
+
+</code-helper>
+The script will check all required actions, which you did in previous steps, and throw the error if something is wrong.
```

Comment:
The script will check all required actions that you completed in the previous steps and will throw an error if something is wrong.